lib/documentLoader.js

```diff
@@ -3,11 +3,10 @@
  */
 import * as didVeresOne from 'did-veres-one';
 import * as ed25519Ctx2018 from 'ed25519-signature-2018-context';
-import ageContext from '@digitalbazaar/age-verification-context';
+import * as ageContext from '@digitalbazaar/age-verification-context';
 import {defaultDocumentLoader} from '@digitalbazaar/vc';
 import {httpClient} from '@digitalbazaar/http-client';
 import {securityLoader} from '@digitalbazaar/security-document-loader';
-import webkmsCtx from '@digitalbazaar/webkms-context';
```

Comment:
linter suggests that `loader.addStatic(webkmsCtx.CONTEXT_URL, webkmsCtx.CONTEXT);` needs to be deleted to.